models/external_login_user.go

```diff
@@ -72,3 +85,94 @@ func removeAllAccountLinks(e Engine, user *User) error {
 	_, err := e.Delete(&ExternalLoginUser{UserID: user.ID})
 	return err
 }
+
+// GetUserIDByExternalUserID get user id according to provider and userID
+func GetUserIDByExternalUserID(provider string, userID string) (int64, error) {
+	var id int64
+	_, err := x.Table("external_login_user").
```

Comment:
Why bypass the ORM? Is there a limitation/advantage? Is it because it's not the PK?

Reply:
`Table("external_login_user")` has a better performance than `Table(new(ExternalLoginUser))`.